packages/financial-templates-lib/src/price-feed/DefiPulsePriceFeed.js

```diff
@@ -15,8 +15,19 @@ class DefiPulseTotalPriceFeed extends PriceFeedInterface {
    * @param {Integer} minTimeBetweenUpdates Min number of seconds between updates. If update() is called again before
    *      this number of seconds has passed, it will be a no-op.
    * @param {Number} priceFeedDecimals Number of priceFeedDecimals to use to convert price to wei.
+   * @param {String} project Project name that we want to query TVL for.
    */
-  constructor(logger, web3, apiKey, lookback, networker, getTime, minTimeBetweenUpdates, priceFeedDecimals = 18) {
+  constructor(
+    logger,
+    web3,
+    apiKey,
+    lookback,
+    networker,
+    getTime,
+    minTimeBetweenUpdates,
+    priceFeedDecimals = 18,
+    project
```

Comment:
Can you validate the value of `project` in the constructor so that the user knows what is acceptable values? For example:

```
constructor(...project) {
const VALID_PROJECTS = ["sushiswap", "uniswap"]
assert(VALID_PROJECTS.includes(project), "Invalid project string");
}
```

This way we won't get potentially obscure errors later when making a request to a URL with a bad `project=` param

Reply:
Added check for project in CreatePriceFeed